models/StringAttribute/StringAttribute.js

```diff
@@ -56,11 +56,17 @@ class StringAttribute {
               size: args.fontSize || 16,
             },
           },
+          MSAttributedStringTextTransformAttribute: textTransformMap[args.textTransform || 'none'],
           MSAttributedStringColorAttribute: new Color(args.color),
           textStyleVerticalAlignmentKey: verticalAlignmentMap[args.verticalAlignment || 'top'],
+          underlineStyle: args.underline ? 1 : 0,
+          strikethroughStyle: args.strikethrough ? 1 : 0,
+          kerning: args.kerning || undefined,
           paragraphStyle: {
             _class: 'paragraphStyle',
             alignment: textAlignmentMap[args.alignment || 'left'],
+            ...(args.lineHeight && { maximumLineHeight: args.lineHeight }),
+            ...(args.lineHeight && { minimumLineHeight: args.lineHeight }),
```

Comment:
may be ?
```javascript
...(args.lineHeight
 ? {
   maximumLineHeight: args.lineHeight,
   minimumLineHeight: args.lineHeight,
 }
 : {}),
```

Reply:
The way @kgoedecke has it is a bit more readable to me, but I'm ok either way.

Reply:
For me to, let @kgoedecke choose

Reply:
Yeah, not the biggest fan of multiline `: ?`. Let's stick with this for now.

Reply:
Ok